src/components/Hero/stories.jsx

```diff
@@ -9,7 +9,7 @@ export default {
 
 export const normal = () =>
     <Hero 
-        image1920x1080="https://staging.core-cms.bfi.digital/sites/default/files/styles/responsive/public/1920/1080/1/2020-03/Hero.jpg"
+        image1920x1080="https://staging.core-cms.bfi.digital/sites/default/files/styles/responsive/public/2020-03/Hero.jpg/1920x1080-cropped/Hero.jpg"
```

Comment:
Out of interest Paul, can we get away with not repeating the filename twice? The `Hero.jpg` bit